odoo/addons/base/models/ir_attachment.py

```diff
@@ -276,7 +276,7 @@ def get_serving_groups(self):
 
     name = fields.Char('Name', required=True)
     description = fields.Text('Description')
-    res_name = fields.Char('Resource Name', compute='_compute_res_name', store=True)
+    res_name = fields.Char('Resource Name', compute='_compute_res_name')
```

Comment:
should probably be removed from https://github.com/odoo/odoo/blob/master/odoo/addons/base/models/ir_attachment.py#L375

Reply:
also the depends don't make sense anymore: https://github.com/odoo/odoo/blob/master/odoo/addons/base/models/ir_attachment.py#L39